README.md

```diff
@@ -27,13 +27,9 @@ npm -g i esy@next
 
 ### macOS
 
-##### XCode
+##### Xcode
 
-Install `Command Line Tools`.
```

Comment:
Having Xcode doesn’t guarantee having command line tools, so we might need both.

Reply:
Ah, gotcha, I'll update the instructions